TSRM/tsrm_win32.c

```diff
@@ -613,6 +614,22 @@ TSRM_API int pclose(FILE *stream)
 #define DESCRIPTOR_PREFIX "TSRM_SHM_DESCRIPTOR:"
 #define INT_MIN_AS_STRING "-2147483648"
 
+
+#define TSRM_BASE_SHM_KEY_ADDRESS 0x20000000
+/* Returns a number between 0x2000_0000 and 0x3fff_ffff. On Windows, key_t is int. */
+static key_t tsrm_choose_random_shm_key(key_t prev_key) {
+	unsigned char buf[4];
+	if (php_win32_get_random_bytes(buf, 4) != SUCCESS) {
+		return prev_key + 2;
```

Comment:
This `2` appears to be an arbitrarily chosen offset; maybe add a respective comment.

Reply:
Guess we can do without. :)